docs/commerce/2.x/en/managing-a-catalog/creating-and-managing-products/channels/introduction-to-channels.md

```diff
@@ -1,22 +1,12 @@
 # Introduction to Channels
 
-Liferay Commerce provides tools designed to meet the needs of complex B2B, multi-region, and multi-market stores and simpler B2C use cases. One way that Commerce enables a wide variety of use cases is through the _Channels_ feature.
+Channels represent a place where products are sold, whether that's a Liferay Commerce based customer portal, a B2B marketplace, or back office phone sales. By connecting sales channels to Liferay Commerce, sellers can manage product content, inventory, orders, and fulfillment across multiple channels in a single place.
 
-In Liferay Commerce, Channels connect Catalogs and Products to any place products are sold: a Liferay Commerce based storefront, third party marketplaces, or back office phone sales. Using Channels, users manage product availability, sales, and fulfillment by Site and eventually by Country, Region, or Account Group. Channels must be associated to warehouses in order to track product availability and and sourcing for product shipping.
+To display products and receive orders, a virtual instance must have at least one channel.
 
-In a simple store example, all of a company's products are managed in one catalog and those products are distributed through a single channel: the company store site.
+With multiple channels, you can create more than one Liferay site each with a unique customer experience, connect to third party software or systems, or a combination of both.
 
-![Simple store structure](./introduction-to-channels/images/01.png)
-
-In a more complex multi-channel scenario, a company may choose to customize the availability and marketing of certain products in its catalog for each of its various channels.
-
-![Multi-channel store structure](./introduction-to-channels/images/02.png)
-
-Liferay Commerce Channels makes managing products, sales, and fulfillment on a single platform simple. See [Configuring Product Visibility by Site Using Channels](./configuring-product-visibility-by-site-using-channels.md) for more information.
-
-![Configuring channels for a product](./introduction-to-channels/images/03.png)
-
-Each channel can be optimized to meet its customers' needs using various scoped features. This includes:
+Products and discounts can be configured to be available in all channels or specific channels. Similarly, many configurations can be set on the channel level. This includes:
```

Comment:
Trying to 1) include a note about managing visibility of a product and scoping discounts to channels, and 2) fix the awkward and confusing phrasing of the existing sentence.